pkg/kp/healthmanager.go

```diff
@@ -230,9 +230,14 @@ func processHealthUpdater(
 			write = nil
 			if result.OK {
 				remoteHealth = result.Health
+				healthLogger := logger.SubLogger(logrus.Fields{
+					"pod":     result.Health.Id,
+					"service": result.Health.Service,
+				})
+				healthLogger.NoFields().Warningf("Service %s is now %s", result.Health.Status, result.Health.Service)
```

Comment:
I think it would be more useful to print this message when the local state changes, instead of when the remote state was successfully updated.  Perhaps do this up around line 210, if the old and new states are not equivalent?